kes.user.js

```diff
@@ -58,7 +59,7 @@
 // ==/UserScript==
 
 //START AUTO MASTHEAD
-/* global addMail, bugReportInit, dropdownEntry, easyEmoticon, hideDownvotes, hideReputation, hideSidebar, hideThumbs, hideUpvotes, initCodeHighlights, initCollapsibleComments, initKFA, initMags, labelOp, magInstanceEntry, mobileHideInit, navbarIcons, notificationsPanel, toggleLogo, updateTime, userInstanceEntry */
+/* global addMail, bugReportInit, dropdownEntry, easyEmoticon, hideDownvotes, hideReputation, hideSidebar, hideThumbs, hideUpvotes, initCodeHighlights, initCollapsibleComments, initKFA, initMags, labelOp, magInstanceEntry, mobileHideInit, navbarIcons, notificationsPanel, toggleLogo, updateTime, userInstanceEntry, textResize */
```

Comment:
Not something that needs to be fixed, but as a general rule going forward, you don't need to modify `kes.user.js` at all, this file is being autogenerated now to include the right script names and other stuff in the masthead. Everything from the top of the file through the line `//END AUTO MASTHEAD` is auto generated. So unless the main logic of the script has to be modified, a mod author doesn't need to submit a change to this file. Mentioning it for the future because it cuts down on the number of apparent changes.